src/components/BreadcrumbView/BreadcrumbView.tsx

```diff
@@ -78,13 +89,22 @@ export class BreadcrumbView extends React.Component<BreadCumbViewProps, BreadCum
     return this.state.pathItem === 'istio';
   };
 
+  isIter8 = () => {
+    return this.state.pathItem === 'iter8';
+  };
```

Comment:
I guess breadcrumb support for extensions is currently explicit?    How would a dev know to add this support, is it documented?